openlibrary/plugins/openlibrary/code.py

```diff
@@ -733,20 +734,13 @@ def most_recent_change():
         return get_recent_changes(limit=1)[0]
 
 
-def wget(url):
-    # TODO: get rid of this, use requests instead.
-    try:
-        return urllib.request.urlopen(url).read()
-    except:
-        return ''
-
 
 @public
 def get_cover_id(key):
     try:
         _, cat, oln = key.split('/')
-        return simplejson.loads(wget('https://covers.openlibrary.org/%s/query?olid=%s&limit=1' % (cat, oln)))[0]
-    except (ValueError, IndexError, TypeError):
+        return requests.get('https://covers.openlibrary.org/%s/query?olid=%s&limit=1' % (cat, oln)).json()[0]
+    except (ValueError, IndexError, TypeError, simplejson.errors.JSONDecodeError):
```

Comment:
Please sort exceptions alphabetically to help the reader spot duplicates and missing exceptions.

`requests` will never raise a simplejson exception but it might have its own JSONDecodeError.